bellman/src/groth16/mod.rs

```diff
@@ -1,10 +1,10 @@
 use group::{CurveAffine, EncodedPoint};
 use pairing::{Engine, PairingCurveAffine};
 
-use SynthesisError;
+use crate::SynthesisError;
 
```

Comment:
Any reason for this gap?

Reply:
Differentiating between imports from local modules and external modules.

Reply:
Sounds good